packer/ubuntu-14.10-i386.json

```diff
@@ -185,7 +185,8 @@
         "scripts/common/vmtools.sh",
         "scripts/common/chef.sh",
         "scripts/ubuntu/cleanup.sh",
-        "scripts/common/minimize.sh"
+        "scripts/common/minimize.sh",
+        "scripts/ubuntu/resolv.sh"
```

Comment:
Why are we editing the config for the 32bit build? AFAIK we're not supporting any 32bit anymore.